### DISCUSSION TITLE
can't see achievement tab or section

### DISCUSSION THREAD

**Discussion by jisaso** [community/community#50503] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/105493394/226374669-dcd2f39e-ab43-4f9e-acbf-d583471e3558.png)

I can see my achievements on the page: https://github.com/jisaso?tab=achievements

But my profile / homepage doesn't show any achievement section or tab.

P.S. I didn't hide any badges, and it's enabled in my profile setting.

**Comment by meppu** [comment]
It looks fine now, github badges and profiles etc... may be updated a little late:
![image](https://user-images.githubusercontent.com/112294217/226636827-1fbdc609-8ccf-4c59-81e3-7ea1062d15d3.png)
